src/services/rest-service.ts

```diff
@@ -204,6 +205,12 @@ const settVarselTilLest = async (varselId: string): Promise<void> => {
     await handleResponse(response);
 };
 
+const setmenu = async (): Promise<string> => {
```

Comment:
`setmenu` høres ut som et endepunkt for å sette menyen. Bør ha et mer beskrivende navn.

Reply:
byttet til sjekkOmBackupmenySkalBrukes()